pkg/master/master.go

```diff
@@ -421,6 +420,25 @@ func (m *Master) init(c *Config) {
 	m.masterServices.Start()
 }
 
+// Adds a service to return the supported api versions.
+func addApiWebService(container *restful.Container, apiPrefix string, enableV1Beta3 bool) {
+	// TODO: InstallREST should register each version automatically
+	versionHandler := apiserver.APIVersionHandler("v1beta1", "v1beta2")
```

Comment:
I think the list is better. That would facilitate moving this back to apiserver.go, if we decide to do that, as well as keeping the decision about which versions are exported in one place.